core-web/apps/dotcms-ui/src/app/portlets/shared/dot-content-types-edit/components/fields/content-type-fields-properties-form/field-properties/dot-relationships-property/dot-relationships-property.component.ts

```diff
@@ -41,6 +41,23 @@ export class DotRelationshipsPropertyComponent implements OnInit {
         this.editing = !!this.group.get(this.property.name).value.velocityVar;
     }
 
+    /**
```

Comment:
Same case of reuse like RegexCheckPropertyComponent, I think is good idea create a class and extend it.

Reply:
That's a pretty cool idea!

I was thinking more about a `.pipe` but now I'm curious about how to achieve that using directives. I think it would be with the `@Host()` decorator to get the form validator from the parent component 🤔. Pretty cool.